Kajabi/sage-lib
# Sage Null View

## Description
Adds new null_view into Sage.

### Screenshots

|  LG Screens  |  SM Screens  |
|--------|--------|
|![Screen Shot 2021-01-27 at 9 28 47 AM](https://user-images.githubusercontent.com/1175111/106029702-3409bb00-6082-11eb-88a1-32f4aee79495.png)|![Screen Shot 2021-01-27 at 9 29 06 AM](https://user-images.githubusercontent.com/1175111/106029748-3d932300-6082-11eb-866c-718e21985af9.png)|

### Steps for testing
1. http://localhost:4000/pages/breakout/object/null_view
2. Verify UI and check object properties.


## Related

- N/A
